data/interfaces/default/base.html

```diff
@@ -20,12 +20,130 @@
     ${next.headIncludes()}
 
     <link rel="icon" type="image/x-icon" href="interfaces/default/images/favicon.ico"/>
-    <!-- touch icons -->
     <link rel="shortcut icon" href="interfaces/default/images/favicon.png">
-    <link rel="apple-touch-icon" href="interfaces/default/images/icon_iphone.png">
-    <link rel="apple-touch-icon" sizes="72x72" href="interfaces/default/images/icon_ipad.png">
-    <link rel="apple-touch-icon" sizes="114x114" href="interfaces/default/images/icon_iphone@2x.png">
-    <link rel="apple-touch-icon" sizes="144x144" href="interfaces/default/images/icon_ipad@2x.png">
+
+    <!-- Allow web app to be run in full-screen mode. -->
+    <meta name="apple-mobile-web-app-capable"
+          content="yes">
+    <!-- Configure the status bar. -->
+    <meta name="apple-mobile-web-app-status-bar-style"
+          content="black">
+    <!-- Set the viewport. -->
+    <meta name="viewport"
+          content="initial-scale=1">
+    <!-- Disable automatic phone number detection. -->
+    <meta name="format-detection"
+          content="telephone=no">
+
+    <!-- ICONS -->
+    <!-- IE10 icon --> 
```

Comment:
Let's also use `xml` for IE. https://msdn.microsoft.com/library/hh772707(v=vs.85).aspx
```
<meta name="application-name" content="PlexPy" />
<meta name="msapplication-config" content="interfaces/default/xml/IEconfig.xml"/>
```
Save this file to `interfaces/default/xml/IEconfig.xml`.
```
<?xml version="1.0" encoding="utf-8"?>
<browserconfig>
  <msapplication>
    <tile>
      <square70x70logo src="interfaces/default/images/res/ie10/tiny.png"/>
      <square150x150logo src="interfaces/default/images/res/ie10/square.png"/>
      <wide310x150logo src="interfaces/default/images/res/ie10/wide.png"/>
      <square310x310logo src="interfaces/default/images/res/ie10/large.png"/>
      <TileColor>#1F1F1F</TileColor>
    </tile>
  </msapplication>
</browserconfig>
```